.github/codecov.yml

```diff
@@ -0,0 +1,8 @@
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        target: auto
+        threshold: 0%
```

Comment:
can the config here be updated to ensure that total coverage never drops below 60% and CI fails if PR reduces coverage by 5%

Reply:
cc @arkodg ，set target: 60%, threshold: 5%  is ok